### PR TITLE
Add an example for multiple callbacks for an active span.

### PR DESCRIPTION
**Pull request by carlosalberto**
Added an example to show how a master task can have a few subtasks, and needs to have its Span closed upon the last one completes.

Haven't been working on OT directly in Java, so let me know in case of any feedback ;)

### CODE REVIEW

File: src/test/java/io/opentracing/contrib/examples/multiple_callbacks/Client.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  public Client(Tracer tracer) {
+    this.tracer = tracer;
+  }
+
+  public void send(final Object message, ActiveSpan parentSpan, final long milliseconds) {
+    final ActiveSpan.Continuation cont = parentSpan.capture();
+    executor.submit(new Callable<Object>() {
+      @Override
+      public Object call() throws Exception {
+        logger.info("Child thread with message '" + message + "' started");
```
Comment (malafeev) [inline src/test/java/io/opentracing/contrib/examples/multiple_callbacks/Client.java]: logger.info("Child thread with message '{}' started", message);

File: src/test/java/io/opentracing/contrib/examples/multiple_callbacks/Client.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        try (ActiveSpan parentSpan = cont.activate()) {
+          try (ActiveSpan span = tracer.buildSpan("subtask").startActive()) {
+            // Simulate work.
+            sleep(milliseconds);
+          }
+        }
+
+        logger.info("Child thread with message '" + message + "' finished");
```
Comment (malafeev) [inline src/test/java/io/opentracing/contrib/examples/multiple_callbacks/Client.java]: logger.info("Child thread with message '{}' finished", message);

File: src/test/java/io/opentracing/contrib/examples/multiple_callbacks/TestMultipleCallbacks.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      Propagator.TEXT_MAP);
+
+  @Test
+  public void test() throws Exception {
+    Client client = new Client(tracer);
+    try (ActiveSpan span = tracer.buildSpan("parent").startActive()) {
+      client.send("task1", span, 100);
+      client.send("task2", span, 200);
+      client.send("task3", span, 300);
+    }
```
Comment (malafeev) [inline src/test/java/io/opentracing/contrib/examples/multiple_callbacks/TestMultipleCallbacks.java]: if we change order 100, 200, 300 to 100, 300, 200 will it be OK?

Comment (carlosalberto) [reply src/test/java/io/opentracing/contrib/examples/multiple_callbacks/TestMultipleCallbacks.java]: Yes, that's ok (just tested it). I was even going to make them sleep randomly, but I wanted the example to be clear about that (reversing their sleep time would still be fine as the user can still see such value).

**Review by malafeev — APPROVED** [review]

**Pull request merged**